projects/storefrontapp-e2e-cypress/package.json

```diff
@@ -18,7 +18,7 @@
     "cy:run:ci:core": "cypress run --config-file cypress.ci.json --record --key $CYPRESS_KEY --tag \"b2c,core\" --parallel --ci-build-id $BUILD_NUMBER --group B2C --spec \"cypress/integration/!(vendor|b2b|ssr|cx_ccv2)/**/*.core-e2e-spec.ts\"",
     "cy:run:ci:b2b": "cypress run --config-file cypress.ci.json --env BASE_SITE=powertools-spa,OCC_PREFIX_USER_ENDPOINT=orgUsers --record --key $CYPRESS_KEY --tag \"b2b,all\" --parallel --ci-build-id $BUILD_NUMBER --group B2B --spec \"cypress/integration/b2b/regression/**/*e2e-spec.ts\" --reporter junit --reporter-options mochaFile=results/e2e-test-b2b-[hash].xml",
     "cy:run:ci:core:b2b": "cypress run --config-file cypress.ci.json --env BASE_SITE=powertools-spa,OCC_PREFIX_USER_ENDPOINT=orgUsers --record --key $CYPRESS_KEY --tag \"b2b,core\" --parallel --ci-build-id $BUILD_NUMBER --group B2B --spec \"cypress/integration/b2b/regression/**/*.core-e2e-spec.ts\"",
-    "cy:run:ci:ccv2": "cypress run --config-file cypress.ci.json --config baseUrl=$ENDPOINT_URL_PUBLIC_SPARTACUS --reporter junit --reporter-options mochaFile=results/spartacus-test-results-[hash].xml --env API_URL=$ENDPOINT_URL_PUBLIC_API --spec \"cypress/integration/cx_ccv2/regression/b2c/**/*.e2e-spec.ts\"",
+    "cy:run:ci:ccv2": "cypress run --config-file cypress.ci.json --config baseUrl=$ENDPOINT_URL_PUBLIC_SPARTACUS --reporter junit --reporter-options mochaFile=results/spartacus-test-results-[hash].xml --env API_URL=$ENDPOINT_URL_PUBLIC_API --spec $E2ES_TO_RUN",
```

Comment:
Just for the sake of discussion: could this command be called directly in `run-spartacus-ccv2-e2es.sh` ?  It doesn't look like `cy:run:ci:ccv2` could be called without $E2ES_TO_RUN defined.

It can totally stay here regardless.

Reply:
related for testing locally here https://github.com/SAP/spartacus/pull/16386#discussion_r1009581502

Moreover, that env variable is made available during the execution of the WTF script